test/pipelineWithInclude.ts

```diff
@@ -16,6 +16,22 @@ test('pipeline with include', () => {
 					ref: 'main',
 					file: '/ci_library/file.yml',
 				},
+				{
+					project: 'somewhere',
+					ref: 'main',
+					file: '/ci_library/file.yml',
+					rules: [
+						{
+							if: 'foo == foo',
+							when: 'never',
```

Comment:
Are you sure `when` is allowed here? I thought `when` was only allowed in job.rules, not include.rules.